cli/validation/converter/iam.go

```diff
@@ -0,0 +1,90 @@
+package converter
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/validation/cai"
+)
+
+// mergeBindings clobbers members to bindings with the same roles
```

Comment:
@ocsig Is this the right behavior in all cases?

Note that in Terraform we have two different merge functions: https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/master/google/iam_helpers.go#L72

The *logic* here is more or less a copy of `mergeAdditiveBindings`, while the comment is reflective of `mergeAuthoritativeBindings`.

Looking at the logic below, I *think* we're likely looking for the "additive" option (ie. not clobbering existing bindings for the same roles).

Reply:
Yes, we need the `mergeAdditiveBindings` functionality which this implementation does. The comment was incorrect, forgot to flag it.